src/base/OpenMCProblemBase.C

```diff
@@ -725,6 +768,15 @@ OpenMCProblemBase::numCells() const
   return n_openmc_cells;
 }
 
+const openmc::Tally &
+OpenMCProblemBase::getKineticsParamTally()
+{
+  if (!_calc_kinetics_params || !_ifp_tally)
```

Comment:
Can this if-statement be equivalently written as `if(!_ifp_tally)`? I guess I'm not clear on when that tally would exist by calc_kinetics_params would be false.